shared/oae/api/oae.api.widget.js

```diff
@@ -275,6 +278,11 @@ define(['exports', 'jquery', 'underscore', 'oae.api.config', 'oae.api.i18n', 'oa
             // The data-widget attribute is removed, to avoid the widget being rendered again
             $element.removeAttr('data-widget');
 
+            // Apply the widget data from the widget loading element on to the widget data for this
+            // widget
+            widgetData[widgetId] = widgetData[widgetId] || {};
+            $.extend(widgetData[widgetId], $element.data());
```

Comment:
How does widget data get explicitly passed in when the widget is loaded from the DOM such as topnav?